### ISSUE TITLE
Argminmax broken on latest nightly when `nightly_simd` is activated

### ISSUE THREAD

**Issue by ritchie46** [jvdd/argminmax#65]
Puts out a whole bunch of:

```
error[E0658]: use of unstable library feature 'stdarch_x86_avx512'
   --> /home/ritchie46/.cargo/registry/src/index.crates.io-6f17d22bba15001f/argminmax-0.6.1/src/lib.rs:1:1
    |
1   | / //! A crate for finding the index of the minimum and maximum values in an array.
2   | | //!
3   | | //! These operations are optimized for speed using [SIMD](https://en.wikipedia.org/wiki/Si...
4   | | //! The SIMD implementation is branchless, ensuring that there is no best case / worst case.
...   |
462 | |                         }
463 | |                     }
    | |___________________________________________________________^
    |
    = note: see issue #111137 <https://github.com/rust-lang/rust/issues/111137> for more information
    = help: add `#![feature(stdarch_x86_avx512)]` to the crate attributes to enable
    = note: this compiler was built on 2024-02-22; consider upgrading it if it is out of date

error[E0658]: use of unstable library feature 'stdarch_x86_avx512'
   --> /home/ritchie46/.cargo/registry/src/index.crates.io-6f17d22bba15001f/argminmax-0.6.1/src/simd/simd_u8.rs:464:53
    |
464 |             let max_index: usize = _mm_extract_epi8(_mm512_castsi512_si128(imin), 0) as usize;
    |                                                     ^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: see issue #111137 <https://github.com/rust-lang/rust/issues/111137> for more information
    = help: add `#![feature(stdarch_x86_avx512)]` to the crate attributes to enable
    = note: this compiler was built on 2024-02-22; consider upgrading it if it is out of date

error[E0635]: unknown feature `stdsimd`
  --> /home/ritchie46/.cargo/registry/src/index.crates.io-6f17d22bba15001f/argminmax-0.6.1/src/lib.rs:72:47
   |
72 | #![cfg_attr(feature = "nightly_simd", feature(stdsimd))]
   |                                               ^^^^^^^

```

**Comment by jvdd** [comment]
Hey @ritchie46,

Thanks for submitting this issue. 
PR #62 fixes this issue (which stems from rust splitting the stdsimd feature into various sub-features). Thus, building the current main branch with the latest nightly rust version should work fine.

I will create a new release containing once #63 is merged into main (which will enable NEON simd in the stable build for AArch64).

Cheers,
Jeroen

**Comment by ritchie46** [comment]
Alright. Thanks for the swift response.

**Comment by jvdd** [comment]
[`v0.6.2`](https://crates.io/crates/argminmax) is now released!
